### PR TITLE
VirHEAT: fix error when no .vcf in samplename

### PR DESCRIPTION
**Pull request by PlushZ**
FOR CONTRIBUTOR:
* [X] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [X] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

**Comment by wm75** [comment]
You need to bump the wrapper version and can you please move the symlinking logic to a configfile as discussed offline?

**Comment by bgruening** [comment]
virheat does not seem to be on bio.tools. Can you maybe add it?

**Comment by PlushZ** [comment]
> virheat does not seem to be on bio.tools. Can you maybe add it?

added https://bio.tools/virheat

**Comment by bgruening** [comment]
And it turned green :)

**Comment by PlushZ** [comment]
I am going to add --reference param to update to 0.7.1 version https://github.com/jonas-fuchs/virHEAT/releases/tag/v.0.7.1

**Comment by PlushZ** [comment]
@bgruening 
ready!

File: tools/virheat/macros.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <macros>
-    <token name="@TOOL_VERSION@">0.7</token>
-    <token name="@VERSION_SUFFIX@">0</token>
+    <token name="@TOOL_VERSION@">0.7.1</token>
+    <token name="@VERSION_SUFFIX@">1</token>
```
Comment (bgruening) [inline tools/virheat/macros.xml:4]:
```suggestion
    <token name="@VERSION_SUFFIX@">0</token>
```

File: tools/virheat/virheat.xml [hunk 2]
```diff
@@ -55,6 +51,20 @@ virheat
         #end if
     #end for
     ]]></command>
+    <configfiles>
+        <configfile filename="vcfs_to_dir.sh"><![CDATA[
+mkdir -p 'vcfs_dir/' &&
```
Comment (bgruening) [inline tools/virheat/virheat.xml:56]: I found the old way easier to read. But it looks fine. Thanks @PlushZ

**Review by bgruening — APPROVED** [review]

**Pull request merged**